### ISSUE TITLE
Review Unfuddle tickets

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/faradays-electromagnetic-lab#8]
This is a legacy (Java) sim, so there are Unfuddle tickets to be reviewed.  Unfuddle tickets should either be closed or migrated to GitHub issues (please link to this issue).

Unfuddle tickets for this sim can be viewed at https://phet.unfuddle.com/a#/projects/9404/ticket_reports/349649

**Comment by pixelzoom** [comment]
There were 16 open Unfuddle tickets.  I reviewed them and created 5 GitHub issues (linked above). @arouinfar you might want to take a quick look at those.

The remaining 11 Unfuddle tickets are all "user request" or "feature request" tickets -- see screenshot below.  I'll leave those to @arouinfar and the sim team to evaluate.

<img width="396" alt="screenshot_2865" src="https://github.com/phetsims/faradays-electromagnetic-lab/assets/3046552/63080015-cb74-4886-b5bb-3bf60753f847">

**Comment by arouinfar** [comment]
@pixelzoom I identified the relevant Unfuddle tickets some time ago and tabulated them here:
https://docs.google.com/document/d/12bQuonZ1hva6zsBQhLQ3pEsiEJmg_rHLyGTY5Ae1J9w/edit#heading=h.l8cjbnb1v89o

I haven't seriously evaluated any of them yet, though.

**Comment by pixelzoom** [comment]
@arouinfar I think it would be useful to parse https://docs.google.com/document/d/12bQuonZ1hva6zsBQhLQ3pEsiEJmg_rHLyGTY5Ae1J9w/edit#heading=h.l8cjbnb1v89o into GitHub issues, so we can more easily track our progress, decisions, etc.

**Comment by pixelzoom** [comment]
For the table in the User Suggestions section of https://docs.google.com/document/d/12bQuonZ1hva6zsBQhLQ3pEsiEJmg_rHLyGTY5Ae1J9w/edit#heading=h.l8cjbnb1v89o, I created GitHub issues for all entries in the table, and noted those GitHub issues in the table.  This will allow us to use GitHub to track our discussions and decisions.

The entries in the table titled "Responses from user suggestions Google Form" still need to be triaged.

**Comment by pixelzoom** [comment]
1/4/24 design meeting with @arouinfar @ariel-phet @kathy-phet @pixelzoom:

We triaged all GitHub issues related to user suggestions.  I made notes in those issues, and closed any associated Unfuddle tickets. There are now zero Unfuddle tickets open for the Java version (project="faraday")

@arouinfar will triage the remaining entries in the "Responses from user suggestions Google Form" table, shown in the design document, bringing anything back to the team that she things needs discussion.
